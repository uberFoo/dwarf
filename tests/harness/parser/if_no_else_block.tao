fn main() {
    if true {} else
}