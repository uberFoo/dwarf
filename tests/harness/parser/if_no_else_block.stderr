[31mError:[0m Unexpected token in input, expected <expression>
   [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0mif_no_else_block:2:16[38;5;246m][0m
   [38;5;246mâ”‚[0m
 [38;5;246m2 â”‚[0m [38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249mi[0m[38;5;249mf[0m[38;5;249m [0m[38;5;249mt[0m[38;5;249mr[0m[38;5;249mu[0m[38;5;249me[0m[38;5;249m [0m[38;5;249m{[0m[38;5;249m}[0m[38;5;249m [0m[31me[0m[31ml[0m[31ms[0m[31me[0m
 [38;5;240m  â”‚[0m                [31mâ”€[0m[31mâ”€[0m[31mâ”¬[0m[31mâ”€[0m  
 [38;5;240m  â”‚[0m                  [31mâ•°[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m Unexpected token [31melse[0m
[38;5;246mâ”€â”€â”€â•¯[0m
[31mError:[0m Unexpected token in input, expected use, struct, Fn, impl
   [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0mif_no_else_block:3:1[38;5;246m][0m
   [38;5;246mâ”‚[0m
 [38;5;246m3 â”‚[0m [31m}[0m
 [38;5;240m  â”‚[0m [31mâ”¬[0m  
 [38;5;240m  â”‚[0m [31mâ•°[0m[31mâ”€[0m[31mâ”€[0m Unexpected token [31m}[0m
[38;5;246mâ”€â”€â”€â•¯[0m

